print/pages/schedule-entry/_id/index.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <v-row no-gutters>
+    <v-col cols="12">
+      <error v-if="$fetchState.error">{{ $fetchState.error.message }}</error>
+      <div v-else-if="!$fetchState.pending">
+        <program-schedule-entry :schedule-entry="scheduleEntry" />
+      </div>
+    </v-col>
+  </v-row>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      config: {},
+      scheduleEntry: null,
+      activity: null,
+      camp: null,
+    }
+  },
+  async fetch() {
+    try {
+      this.scheduleEntry = await this.$api
+        .get()
+        .scheduleEntries({ scheduleEntryId: this.$route.params.id })._meta.load
+
+      this.activity = await this.scheduleEntry.activity()._meta.load
+      this.camp = await this.activity.camp()._meta.load
+    } catch (e) {
+      this.$nuxt.context.res.statusCode = 404
+      throw new Error('Schedule Entry not found')
+    }
+  },
+  head() {
+    const header = {}
+
+    /**
+     * Footer & header for single activity view
+     */
+    header.__dangerouslyDisableSanitizers = ['style'] // disable sanitzing of below inline css
```

Comment:
Auch hier, verschieben in unscoped `<style>` Block?

Reply:
Siehe Kommentar https://github.com/ecamp/ecamp3/pull/1341#discussion_r628701226

Im `<style>` block habe ich kein `${this.camp.name}`. Was ich mir machen sollte ist sanitizing des camp-names.